src/libraries/Common/src/Internal/Cryptography/UniversalCryptoTransform.cs

```diff
@@ -17,7 +17,9 @@ namespace Internal.Cryptography
     //
     internal abstract class UniversalCryptoTransform : ICryptoTransform
     {
-        public static ICryptoTransform Create(PaddingMode paddingMode, BasicSymmetricCipher cipher, bool encrypting)
+        public static ICryptoTransform Create(PaddingMode paddingMode,
+                                              BasicSymmetricCipher cipher,
+                                              bool encrypting)
```

Comment:
The prevailing style in crypto (if not all of the libraries code) is to chop before the first parameter and indent once.  Though it could also be left on just the one line.

```suggestion
        public static ICryptoTransform Create(
            PaddingMode paddingMode,
            BasicSymmetricCipher cipher,
            bool encrypting)
```